hibernate-core/hibernate-core.gradle

```diff
@@ -55,6 +43,15 @@ dependencies {
     compile( libraries.commons_annotations )
     antlr( libraries.antlr )
 
+    // JAXB
+    compile( libraries.jaxb_api )
+    compile( libraries.jaxb_runtime )
```

Comment:
This is a significant change and we need to be sure it's handled well in WildFly. Do they include the correct JAXB versions in all cases (e.g. even for JDK 8, now that we compile against the latest versions even for JDK 8)?

Reply:
wildfly includes JAXB 2.3 SPEC API + JAXB impl (runtime) jar that works with 2.3 (basically the JAXB version that aligns with Java EE 8 + Jakarta EE 8, which are equivalent).

Reply:
OK, perfect, thanks Scott.